flow/src/benchmarking.rs

```diff
@@ -1,16 +1,194 @@
 //! Benchmarking setup for gamedao-flow
 use super::*;
 
-#[allow(unused)]
 use crate::Pallet as Flow;
-use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite, whitelisted_caller};
 use frame_system::RawOrigin;
+use frame_support::traits::Hooks;
+use sp_runtime::{DispatchError, traits::{Saturating, SaturatedConversion}};
 use sp_std::vec;
 
+
+const SEED: u32 = 0;
+const DEPOSIT_AMOUNT: u128 = 10000000000000000000;
```

Comment:
```suggestion
const DEPOSIT_AMOUNT: u128 = 10_000_000_000_000_000_000;
```
Or `DOLLARS`, but I understand it's more lines.